fixes.inc

```diff
@@ -6308,6 +6382,24 @@ native BAD_TogglePlayerControllable(playerid, toggle) = TogglePlayerControllable
 		//  END: OnPlayerEnterVehicle_2
 		// =============================
 
+		// ===============================
+		//  BEGIN: OnPlayerEnterVehicle_3
+		// ===============================
+		#if FIX_OnPlayerEnterVehicle_3
+			if (FIXES_gsVehicleIsLocked[vehicleid]) {
```

Comment:
Yeah sorry I do that too in my project, I think it's more visible.
I copied this line from a sloppy programmer https://github.com/ziggi/samp-enterfix/blob/master/enterfix.inc#L34 sorry!
https://github.com/Open-GTO/sa-mp-fixes/pull/64/commits/e12925e4f15cfd10bf2081e1cd605584dae09157